src/app/utils/audio.ts

```diff
@@ -0,0 +1,4 @@
+export const music = document.addEventListener("DOMContentLoaded", event => {
+    const audio = document.querySelector("audio");
+    audio.volume = 0.7;
```

Comment:
There's an error. You don't specify type of audio variable. You can fix it by adding 'as HTMLAudioElement' at the end of line 2.